src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue

```diff
@@ -116,24 +128,54 @@ export default defineComponent<DashboardBoardListProps>({
                     }
                 ))),
         });
+        const deleteModalState = reactive({
+            headerTitle: i18n.t('Are you sure you want to delete dashboard?'),
+            visible: false,
+            loading: false,
+            selectedId: undefined,
```

Comment:
nitpick: Please add type assertion

```suggestion
            selectedId: undefined as string|undefined,
```